MapboxVisionTests/VisionManagerTests.swift

```diff
@@ -61,4 +61,62 @@ class VisionManagerTests: XCTestCase {
             "VisionManager should be successfully deallocated after calling destroy and releasing the object"
         )
     }
+
+    func testPerformanceConfigMergedTranslatesToSamePerformance() {
+        // Given
+        // object of VisionManager
+
+        // When
+        // model performance config is set
+        let performance = ModelPerformance(mode: .dynamic, rate: .high)
+        visionManager.modelPerformanceConfig = .merged(performance: performance)
+
+        // Then
+        XCTAssertEqual(visionManager.modelPerformance, performance)
+    }
+
+    func testPerformanceConfigSeparateDynamicModeTranslatesToHigherRate() {
+        // Given
+        // object of VisionManager
+
+        // When
+        // model performance config is set
+        let performance1 = ModelPerformance(mode: .dynamic, rate: .low)
+        let performance2 = ModelPerformance(mode: .dynamic, rate: .high)
+        visionManager.modelPerformanceConfig = .separate(segmentationPerformance: performance1,
+                                                         detectionPerformance: performance2)
+
+        // Then
+        XCTAssertEqual(visionManager.modelPerformance, performance2)
+    }
+
+    func testPerformanceConfigSeparateFixedModeTranslatesToHigherRate() {
+        // Given
+        // object of VisionManager
+
+        // When
+        // model performance config is set
+        let performance1 = ModelPerformance(mode: .fixed, rate: .low)
+        let performance2 = ModelPerformance(mode: .fixed, rate: .high)
+        visionManager.modelPerformanceConfig = .separate(segmentationPerformance: performance1,
+                                                         detectionPerformance: performance2)
+
+        // Then
+        XCTAssertEqual(visionManager.modelPerformance, performance2)
+    }
+
+    func testPerformanceConfigSeparateDynamicAndFixedModesTranslatesToFixedMode() {
+        // Given
+        // object of VisionManager
+
+        // When
+        // model performance config is set
+        let performance1 = ModelPerformance(mode: .dynamic, rate: .low)
+        let performance2 = ModelPerformance(mode: .fixed, rate: .high)
+        visionManager.modelPerformanceConfig = .separate(segmentationPerformance: performance1,
+                                                         detectionPerformance: performance2)
+
+        // Then
+        XCTAssertEqual(visionManager.modelPerformance, performance2)
+    }
```

Comment:
The tests above check the correctness of deprecated interfaces.

Should we add tests for the new interfaces (comparators, `coreModelPerformance`, `modelPerformance`) as well?

Reply:
All the mentioned API translates to very basic functionality like one comparison or one set of a value. I would omit tests for them as redundant. WDYT?

Reply:
Agree.